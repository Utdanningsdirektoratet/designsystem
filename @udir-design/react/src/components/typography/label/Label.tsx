import {
  Label as DigdirLabel,
  type LabelProps as DigdirLabelProps,
} from '@digdir/designsystemet-react';
import { ComponentRef, ForwardRefExoticComponent, RefAttributes } from 'react';

type LabelProps = Omit<DigdirLabelProps, 'data-color'>;

const Label = DigdirLabel as ForwardRefExoticComponent<
  LabelProps & RefAttributes<ComponentRef<typeof DigdirLabel>>
>;

export { Label, LabelProps };
