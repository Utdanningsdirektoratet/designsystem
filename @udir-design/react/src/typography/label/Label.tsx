export { Label, type LabelProps } from '@digdir/designsystemet-react';
