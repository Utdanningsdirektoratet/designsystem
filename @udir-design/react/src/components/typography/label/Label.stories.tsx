import type { Meta, StoryObj } from '@storybook/react-vite';
import { Label } from './Label';

const meta: Meta<typeof Label> = {
  component: Label,
  tags: ['beta', '!autodocs', 'digdir'],
  parameters: {
    componentOrigin: {
      originator: 'digdir',
    },
  },
  title: 'Components/Typography/Label',
};

export default meta;
type Story = StoryObj<typeof Label>;

export const Preview: Story = {
  args: {
    children: 'Vennligst skriv inn f√∏dselsnummer. 11 tegn',
    weight: 'medium',
  },
};
