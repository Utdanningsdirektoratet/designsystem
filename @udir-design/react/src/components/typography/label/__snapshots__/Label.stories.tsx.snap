// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Preview 1`] = `
<label data-weight="medium">
  Vennligst skriv inn f√∏dselsnummer. 11 tegn
</label>
`;
